luigi/contrib/spark.py

```diff
@@ -378,6 +378,9 @@ def run(self):
         py_files = self.py_files()
         if py_files != []:
             options += ['--py-files', ','.join(py_files)]
+        dependency_jars = self.dependency_jars()
+        if dependency_jars != []:
```

Comment:
Nitpick: I see that `xxx != []` is the current convention in this file (see line 379 for example), but I'd prefer `if dependency_jars:`.